### PR TITLE
Add more icons

### PR DESCRIPTION
**Pull request by dvdmrtnz**
* Add emergency=assembly_point. Closes https://github.com/organicmaps/organicmaps/issues/4954

| Day | Night |
|-|-|
|  | 

* Add tourism=aquarium. Closes https://github.com/organicmaps/organicmaps/issues/3604

| Day | Night |
|-|-|
|  | 

* Fixed barrier=gate color

* Fixed shop=sewing color

### CODE REVIEW

**Comment by Jean-BaptisteC** [comment]
POI emergency in Organic Maps is red ?

**Comment by dvdmrtnz** [comment]
> POI emergency in Organic Maps is red ?

Red assembly point 

IMG_2697
IMG_2695

File: data/categories.txt [hunk 1]
```diff
@@ -7118,6 +7118,45 @@ sw:Mbuga
 fa:پارک
 mr:उद्यान|पार्क
 
+tourism-aquarium
```
Comment (pastk) [inline data/categories.txt:7121]: `tourism-aquarium|@tourism` ?
to add into the "Sights" search category

**Review by pastk — APPROVED** [review]
Looks great, thanks!

File: data/categories.txt (hunk 1 above)
Comment (dvdmrtnz) [reply data/categories.txt]: Added

**Review by vng — APPROVED** [review]

**Pull request merged**